test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -109,7 +109,8 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 		onlineCPUSet, err = nodes.GetOnlineCPUsSet(ctx, workerRTNode)
 		Expect(err).ToNot(HaveOccurred())
 		cpuID := onlineCPUSet.UnsortedList()[0]
-		smtLevel = nodes.GetSMTLevel(ctx, cpuID, workerRTNode)
+		smtLevel, err = nodes.GetSMTLevel(ctx, cpuID, workerRTNode)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
Here, we can error with message so that it's clear 
`Expect(err).ToNot(HaveOccurred(), "Unable to fetch  SMT level on node %s, Error: %v", workerRTNode.Name, err)`

Reply:
same for others also.